controls/sshd_spec.rb

```diff
@@ -77,7 +77,6 @@
     it { should exist }
     it { should be_file }
     it { should be_owned_by 'root' }
-    it { should be_grouped_into 'root' }
```

Comment:
@techraf you don't have to apologize, there is no reason :) 

Could you also please make [rubocop happy](https://travis-ci.org/dev-sec/ssh-baseline/jobs/192667173)?